hooks/pre-exit

```diff
@@ -40,6 +43,52 @@ if [[ -z "${FORMAT}" ]]; then
   exit 1
 fi
 
+# Creates the annotations with all the report URLs
+annotation-link() {
+  local json_file="$1"
+  mapfile -t REPORT_URLS < <(sort "${json_file}" | uniq)
+
+  local REPORTS=""
+  REPORT_COUNT=1
+  if [ ${#REPORT_URLS[@]} -gt 0 ]; then
+    echo "Got ${#REPORT_URLS[@]} report URLs."
+    for URL in "${!REPORT_URLS[@]}"; do
```

Comment:
This bash construct returns the indexes of the array you created, not the values, so `URL` will have the values 0, 1, ... etc.

```suggestion
    for URL in "${REPORT_URLS[@]}"; do
```